src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -10,32 +10,49 @@
      * ставка за 1 час. Метод должен вернуть отчёт за период, который передали в метод
      * (обе даты включительно) составленный по следующей форме: Отчёт за период
      * #дата_1# - #дата_2# Имя сотрудника - сумма заработанных средств за этот период
-     *
      * Пример ввода: date from = 01.04.2019 date to = 30.04.2019
-     *
      * names:
      * Сергей
      * Андрей
      * София
-     *
      * data:
      * 26.04.2019 Сергей 60 50
      * 26.04.2019 Андрей 3 200
-     * 26.04.2019 Сергей 7 100
-     * 26.04.2019 София 9 100
-     * 26.04.2019 Сергей 11 50
-     * 26.04.2019 Андрей 3 200
-     * 26.04.2019 Сергей 7 100
-     * 26.04.2019 София 9 100
-     * 26.04.2019 Сергей 11 50
-     *
+     * 27.04.2019 Сергей 7 100
+     * 28.04.2019 София 9 100
+     * 28.04.2019 Сергей 11 50
+     * 28.04.2019 Андрей 3 200
+     * 29.04.2019 Сергей 7 100
+     * 29.04.2019 София 9 100
+     * 30.04.2019 Сергей 11 50
      * Пример вывода:
      * Отчёт за период 01.04.2019  - 30.04.2019
      * Сергей - 1550
      * Андрей - 600
      * София - 900
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        if (Integer.valueOf(dateFrom.replaceAll("[^0-9]",""))
```

Comment:
what if you will compare dateFrom = 14.07.2019 and dateTo = 10.08.2019?

Reply:
agreed, but there was no such a condition!

Reply:
fixed